package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsonata",
-  "version": "1.5.4",
+  "version": "1.5.5",
   "description": "JSON query and transformation language",
```

Comment:
Could you please not bump the version in this commit. We usually prefer to do the release separately from the change so we can bundle things up (Travis will automatically publish to npm on a new version number)